util/settings/accounts_test.go

```diff
@@ -0,0 +1,67 @@
+package settings
```

Comment:
General thought on this test package. I think we should also test for bad/malformed input data, that would also come up with more code coverage.

Reply:
Done. Added missing tests